integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.78 Version bump reason: [Tizen] Fix environment variable order, add missing tizen dependencies
+0.5.79 Version bump reason: [Ameba] To adapt the 'reset-watermarks' and 'read thread-metrics' functions added in Matter sdk
```

Comment:
what does "to adapt" mean? 

Could this be for #19147 ? In which case I assume this should rather be "update ameba sdk to expose xPortResetHeapMinimumEverFreeHeapSize freeRTOS API" or something similar.

Reply:
I have changed the description.